scripts/src/checkautomerge/checkautomerge.py

```diff
@@ -1,18 +1,25 @@
 import time
 import sys
 import argparse
+import os
 
 import requests
 
 def ensure_pull_request_not_merged(api_url):
     # api_url https://api.github.com/repos/<organization-name>/<repository-name>/pulls/1
-    headers = {'Accept': 'application/vnd.github.v3+json'}
+    headers = {'Accept': 'application/vnd.github.v3+json','Authorization': f'Bearer {os.environ.get("BOT_TOKEN")}'}
     merged = False
     for i in range(20):
         r = requests.get(api_url, headers=headers)
-        if r.json()["merged"]:
+        response_content = r.json()
+        if "message" in response_content:
+            print(f'[ERROR] getting public key: {response_content["message"]}')
```

Comment:
I think you meant `[Error] getting merge status` here

Reply:
Thx, updated.